spec/models/chat_channel_spec.rb

```diff
@@ -78,7 +78,7 @@
 
       expect(events).to include(
         event_name: :chat_channel_status_change,
-        params: [{ channel: private_category_channel, old_status: :open, new_status: :closed }],
+        params: [{ channel: private_category_channel, old_status: "open", new_status: "closed" }],
```

Comment:
I assumed having strings instead of symbols for event params won’t matter much. If that’s not the case, just tell me :slightly_smiling_face: